src/liblibc/lib.rs

```diff
@@ -1523,19 +1543,15 @@ pub mod types {
                     pub st_uid: uid_t,
                     pub st_gid: gid_t,
                     pub st_rdev: dev_t,
-                    pub st_atime: time_t,
-                    pub st_atime_nsec: c_long,
-                    pub st_mtime: time_t,
-                    pub st_mtime_nsec: c_long,
-                    pub st_ctime: time_t,
-                    pub st_ctime_nsec: c_long,
+                    pub st_atim: timespec,
+                    pub st_mtim: timespec,
+                    pub st_ctim: timespec,
                     pub st_size: off_t,
```

Comment:
I am not completely agreed with the conversion form `time_t + c_long` to `timespec`.
Under bitrig and openbsd, the timespec definition is under conditionnal definition:
`#if __POSIX_VISIBLE >= 200809 || __BSD_VISIBLE`. Here, the mod is `posix01`.

Note: the C size of `timespec` is the same as `time_t + c_long`, so it don't break code, only the ABI.

see https://github.com/bitrig/bitrig/blob/master/sys/sys/stat.h#L53